packages/material-ui/src/Modal/Modal.js

```diff
@@ -288,7 +289,7 @@ Modal.propTypes = {
   /**
    * A single child content element.
    */
-  children: PropTypes.element.isRequired,
+  children: elementAcceptingRef.isRequired,
```

Comment:
This is actually only true for `TrapFocus`. But since `Modal` always uses this component and `TrapFocus` is an implementation detail we warn at the `Modal` boundary.